pages/linux/zile.md

```diff
@@ -0,0 +1,28 @@
+# zile
+
+> Zile is a lightweight clone of the Emacs text editor.
+> More information: <https://www.gnu.org/software/zile/>.
+
+- Start in console mode (without X window):
+
+`zile -nw`
```

Comment:
Why I'm getting this message `Unknown option '-nw'`?

Reply:
Zile is a console only editor. Hence, -w was redundant. Have removed it now.

Reply:
I tried `-n` too but: `Unknown option '-n'`

Reply:
Have removed them both now. Apologies for the silly error.